doc/comandi/converti-opzioni.md

```diff
@@ -1,54 +1,36 @@
 
 ## Opzioni di `converti`
 
-Elenchiamo subito le opzioni. Dopo l'elenco trovate una spiegazione
-sul come usarle
+Di seguito è possibile trovare l'elenco delle opzioni di 'converti' e una breve descrizione
 
-### `collegamento-normativa`
+### Tutte le opzioni di 'converti' sono  suscettibili di errori di sintassi rST, pertanto i risultati andranno verificati. 
 
-Quando attivate questa opzione, `converti` prova a trasformare tutti i
-riferimenti a leggi in collegamenti ipertestuali a Normattiva. Lo
-svantaggio è che il processo potrebbe introdurre errori di sintassi
-rST
+### `link-normattiva`
```

Comment:
Perché _link_? Non ha più senso mantenere tutto in Italiano?

Reply:
link o collegamento non fa la differenza, l'idea era quello di sostituire normativa con normattiva. tutto qui! :-)

Reply:
ah okay, mi sembra una buona idea!